MonoGame.Framework.Content.Pipeline/Audio/AudioContent.cs

```diff
@@ -281,6 +281,10 @@ public void ConvertFormat(ConversionFormat formatType, ConversionQuality quality
                     channelCount,
                     format,
                     sampleRate);
+
+                // Loop start and length in number of samples. Defaults to entire sound
+                loopStart = 0;
+                loopLength = data.Count / ((bitsPerSample / 8) * channelCount);
```

Comment:
It was returning zero, which I assume XAudio takes as meaning the whole file.  That is why the MonoGame-processed xnb appeared to work.